spec/factories/miq_dialog.rb

```diff
@@ -37,4 +37,14 @@
     name        "miq_provision_configured_system_foreman_dialogs"
     dialog_type "MiqProvisionConfiguredSystemWorkflow"
   end
+
+  factory :miq_dialog_aws_provision, :parent => :miq_dialog do
+    name        "miq_provision_amazon_dialogs_template"
+    dialog_type "MiqProvisionWorkflow"
+
+    content do
```

Comment:
it is not valid unless I create the provision dialog with actual values like the yaml has.
do you mean moving the yaml loading part in tests ? I think I can